.gitignore

```diff
@@ -3,7 +3,8 @@
 .classpath
 .DS_Store
 .factorypath
-.idea
+**/.idea/*
```

Comment:
How does this apply to this PR / under which circumstances may this happen? :)

Would expect that this suffices: 
```suggestion
**/.idea
```

Reply:
I did this as I got some changes in the `integration-test` directory of Checkstyle and the `.idea` folder. I made this change and it went away.

Reply:
The `/*` part was required as we have the inclusion afterwards.